bindings/cs/MongoDB.Libmongocrypt/MongoDB.Libmongocrypt.targets

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <ItemGroup Condition="'$(OS)' == 'Windows_NT'">
+    <ItemGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
```

Comment:
See discussion in [slack](https://mongodb.slack.com/archives/CB2AM9G13/p1567544025177300)